arangod/Graph/ShortestPathOptions.cpp

```diff
@@ -261,6 +279,19 @@ ShortestPathOptions::ShortestPathOptions(ShortestPathOptions const& other,
   TRI_ASSERT(other._defaultWeight >= 0.);
 }
 
+#ifndef USE_ENTERPRISE
+auto ShortestPathOptions::setDisjoint() -> void { return; }
+
+auto ShortestPathOptions::isDisjoint() const -> bool { return false; }
+
+auto ShortestPathOptions::isSatelliteLeader() const -> bool {
+  // Can only be called in Enterprise code.
+  // Return false as security net.
+  TRI_ASSERT(false);
+  return false;
+}
+#endif
```

Comment:
Ideally these should be defined only in the enterprise case, and hence not need dummy implementations for the community case?

Reply:
The compiler needs them to be implemented, otherwise the build will fail for CE.

Reply:
Currently that's the way it is used to be. I would like to think about a better way as soon as we clean GraphNodes(TraversalNode ..) + Options.